app/controllers/user_journey_controller.rb

```diff
@@ -0,0 +1,69 @@
+class UserJourneyController < ApplicationController
+  layout "main_layout"
+  include ControllerConcern
+  include ComponentConcern
+  include CertificateConcern
+
+  def index
+    @sp_components = SpComponent.all
+    @msa_components = MsaComponent.all
+  end
+
+  def view_certificate
+    @certificate = Certificate.find_by_id(params[:certificate_id])
+  end
+
+  def before_you_start
+    @certificate = Certificate.find_by_id(params[:certificate_id])
```

Comment:
this line is repeated in pretty much each method, so it might be worth pulling it as a function (before each method maybe?) and add some validation/error handling along with it. But again, happy to make it a part of a follow-up PR